<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generador de dialogos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Titulo de la ventana:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mensaje:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Iconos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>471</width>
      <height>142</height>
     </rect>
    </property>
    <property name="title">
     <string>Icono:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="btn_ninguno">
       <property name="text">
        <string>Ninguno</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_informacion">
       <property name="text">
        <string>Información</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_pregunta">
       <property name="text">
        <string>Pregunta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_precaucion">
       <property name="text">
        <string>Precaución</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_critico">
       <property name="text">
        <string>Critico</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="botones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Botones:</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="4">
      <widget class="QCheckBox" name="ch_ignorar">
       <property name="text">
        <string>Ignorar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="ch_abortar">
       <property name="text">
        <string>Abortar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QCheckBox" name="ch_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="ch_si">
       <property name="text">
        <string>Si</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="ch_reintentar">
       <property name="text">
        <string>Reintentar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="ch_no">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="ch_cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="ch_cerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="ch_abrir">
       <property name="text">
        <string>Abrir</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="ch_guardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QCheckBox" name="ch_guardar_todo">
       <property name="text">
        <string>Guardar todo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="mostrar_mensaje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>471</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar mensaje</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titulo_ventana">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mensaje">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
